packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1193,7 +1199,7 @@ export class DeltaManager
 
         if (this.closed) {
             // Raise proper events, Log telemetry event and close connection.
-            this.disconnectFromDeltaStream(`Disconnect on close`);
+            this.disconnectFromDeltaStream("Disconnect new connection since closed");
```

Comment:
It was a bit hard for me to parse that message.
I'd drop "new connection" because it does not carry much value - we close connections on any closure, no matter what's the order of events. I'd prefer to use same message as in close() method - which is (in this PR) - "Closing DeltaManager"

Reply:
Ok, interesting.  I guess it's basically an uninteresting race condition to end up here v. there?

Reply:
Yes, I do not think there is any difference RE how we look at these events. Maybe the only difference - the connect can come through a minute after closure, and then the right question to ask - should we built cancelation into connection process? But I know the answer without data (yes, we should :)), so it's just a matter of priority, I guess.

Reply:
My gut is it will be nice to have a separate reason for this case.  Going with "DeltaManager already closed"